<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockMarketDialog</class>
 <widget class="QDialog" name="StockMarketDialog">
  <property name="geometry"><rect><x>0</x><y>0</y><width>500</width><height>400</height></rect></property>
  <property name="windowTitle"><string>Stock Market</string></property>
  <widget class="QWidget" name="widget">
   <widget class="QTextEdit" name="stockTextEdit"><property name="geometry"><rect><x>20</x><y>20</y><width>460</width><height>200</height></rect></property><property name="readOnly"><bool>true</bool></property></widget>
   <widget class="QSpinBox" name="accountIdSpinBox"><property name="geometry"><rect><x>20</x><y>240</y><width>100</width><height>30</height></rect></property></widget>
   <widget class="QLineEdit" name="symbolLineEdit"><property name="geometry"><rect><x>140</x><y>240</y><width>100</width><height>30</height></rect></property></widget>
   <widget class="QSpinBox" name="amountSpinBox"><property name="geometry"><rect><x>260</x><y>240</y><width>60</width><height>30</height></rect></property></widget>
   <widget class="QPushButton" name="buyButton"><property name="geometry"><rect><x>340</x><y>230</y><width>60</width><height>40</height></rect></property><property name="text"><string>Buy</string></property></widget>
   <widget class="QPushButton" name="sellButton"><property name="geometry"><rect><x>420</x><y>230</y><width>60</width><height>40</height></rect></property><property name="text"><string>Sell</string></property></widget>
   <widget class="QPushButton" name="backButton"><property name="geometry"><rect><x>20</x><y>300</y><width>100</width><height>30</height></rect></property><property name="text"><string>Back</string></property></widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
